environment/src/main/java/com/sequenceiq/environment/environment/flow/deletion/handler/sdx/DataLakeClustersDeleteHandler.java

```diff
@@ -52,11 +53,11 @@ public void accept(Event<EnvironmentDeletionDto> environmentDtoEvent) {
 
         try {
             PollingConfig pollingConfig = getPollingConfig();
-            environmentService.findEnvironmentById(environmentDto.getId())
-                    .ifPresent(environment -> sdxDeleteService.deleteSdxClustersForEnvironment(
+            EnvironmentView environment = environmentViewService.getById(environmentDto.getId());
+            sdxDeleteService.deleteSdxClustersForEnvironment(
```

Comment:
Going down `sdxDeleteService.deleteSdxClustersForEnvironment(...)` I saw that it only used the `name` and the `resouceCrn` so way less than what is requested from the database. I don't think this is a bottleneck but worths mentioning.

Reply:
True, I'm planning to eliminate these, but this PR is so huge and nobody wants to merge